server/utils/helpers/chat/convertTo.js

```diff
@@ -33,10 +35,27 @@ async function convertToJSONL(workspaceChatsMap) {
     .join("\n");
 }
 
-async function prepareWorkspaceChatsForExport() {
+async function prepareWorkspaceChatsForExport(type = "jsonl") {
   const chats = await WorkspaceChats.whereWithData({}, null, null, {
     id: "asc",
   });
+
+  if (type === "csv") {
+    const preparedData = chats.map((chat) => {
+      const responseJson = JSON.parse(chat.response);
+      return {
+        id: chat.id,
+        username: chat.user ? chat.user.username : "unknown user",
+        workspace: chat.workspace ? chat.workspace.name : "unknown workspace",
+        prompt: chat.prompt,
+        response: responseJson.text,
+        sent_at: chat.createdAt,
+      };
+    });
+
+    return preparedData;
+  }
```

Comment:
im not crazy about adding this here but I cant think of better way to do this or position it